tests/wait-for.test.ts

```diff
@@ -40,7 +63,9 @@ test(
       tests/wait-for.test.ts
 
               throw new Error('something bad happened');
-                    ^"
+                    ^
+      -------------------------------------------------------
+      dist/cjs/index.cjs"
```

Comment:
Just double checking that it was expected for this test's output to change as well.

Reply:
I am not entirely sure why this changed (not sure why it was how it was before this PR) but yes this is the expected output